src/cmds/qsub.c

```diff
@@ -1314,7 +1314,7 @@ strdup_esc_commas(char *str_to_dup)
 		while (*roaming!='\0' && *roaming!=',')
 			*(endstr++)=*(roaming++);
 		if (*roaming==',') {
-			*(endstr++)='\\';
+			*(endstr++)=ESC_CHAR;
```

Comment:
Please add space around '=' and would be great if you can fix the three lines above as well : )

Reply:
I fixed all these cases inside the function. It's an old code, strange style it had.